pom.xml

```diff
@@ -473,6 +473,10 @@
 								<requireMavenVersion>
 									<version>[3.1.1,)</version>
 								</requireMavenVersion>
+								<requireJavaVersion>
+									<!-- FIXME: remove the upper bound once we migrated to Spring 5 -->
+									<version>[11,19)</version>
```

Comment:
Excluding Java 19 might not be necessary for building, but definitely for running since it does not support the `--add-opens` workaround.